Resources/AzureDevOps/azure-pipelines.yml

```diff
@@ -51,12 +58,27 @@ stages:
       ParentBusiness: '$(ParentBusiness)'
       ServiceOffering: '$(ServiceOffering)'
       ResourceGroup: 'dfc-dev-stax-api-rg'
-      WebAppPrefix: 'dfc-dev'
       ArtifactRootDirectory: "$(System.ArtifactsDirectory)/DFC.ServiceTaxonomy.ApiFunction'"
       PackageFile:  "$(System.ArtifactsDirectory)/DFC.ServiceTaxonomy.ApiFunction/DeploymentPackages/DFC.ServiceTaxonomy.ApiFunction.zip"
       ArmTemplateFolder: "$(System.ArtifactsDirectory)/DFC.ServiceTaxonomy.ApiFunction/Azure/ArmTemplates/"
       ApimImportScript: "$(System.ArtifactsDirectory)/DFC.ServiceTaxonomy.ApiFunction/Azure/Scripts/Import-ApimOpenApiDefinitionFromFile.ps1"
       SpecificationPath: "$(System.ArtifactsDirectory)/DFC.ServiceTaxonomy.ApiFunction/OpenApiDocs"
       SharedResourceGroup: 'dfc-dev-shared-rg'
       ApimServiceName: 'dfc-dev-shared-apim'
-      FunctionAppDomain: '$(FunctionAppDomain)'
\ No newline at end of file
+      FunctionAppDomain: '$(FunctionAppDomain)'
+      EnvironmentName: DEV_SERVICETAXONOMY
+      # The Apis parameter requires a complex object be passed in,  which requires some odd looking yaml.
+      # The top level -`s here defines an array of "something"
+      # The inner level -'s defines an array of mappings.
+      Apis: 
+        -
+          - name: GetAllSkills
+            azureFunctionName: "${{ variables.WebAppPrefix }}-stax-skills-fa"
+          - name: GetAllOccupations
+            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-occ-fa'
+          - name: GetAllSkillsForOccupation
+            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-skillsocc-fa'
+        -
+          - name: GetOccupationsByLabel
+            azureFunctionName: '${{ variables.WebAppPrefix }}-stax-occbylabel-fa'
+            dependsOn: GetAllSkills
```

Comment:
Should be a comment to make it explicit what the dependsOn is for.   "dependsOn properties are set on later array members to control the number of deployment jobs that run in parallel"